src/test/java/net/sf/jabref/exporter/ExportFormatTest.java

```diff
@@ -23,75 +24,68 @@
 import net.sf.jabref.model.database.BibDatabase;
 import net.sf.jabref.model.entry.BibEntry;
 
+@RunWith(Parameterized.class)
 public class ExportFormatTest {
 
+    public IExportFormat exportFormat;
+    public String exportFormatName;
+    public static BibDatabase database;
+    public static Charset charset;
+    public static MetaData metaData;
+    public static List<BibEntry> entries;
+
+    public ExportFormatTest(IExportFormat format, String name) {
+        exportFormat = format;
+        exportFormatName = name;
+    }
+
+
     @Rule
     public TemporaryFolder testFolder = new TemporaryFolder();
 
 
-    @Before
-    public void setUp() {
-        Globals.prefs = JabRefPreferences.getInstance();
-        ExportFormats.initAllExports();
+    @BeforeClass
```

Comment:
And regarding providing an actual database, the current method is not the best approach (with statics). However, it seems like the tests needs to be rather specific anyway for the different export formats so maybe they can't be parameterized in a good way anyhow.